StreamConsumer/README.md

```diff
@@ -352,7 +352,7 @@ Rapid pro requires a specific format for phone numbers and it's better to change
 
 ```json
 {
-    "id": "rural_3",
+    "id": "rural_3_2",
```

Comment:
@shawnsarwar, incrementing the pipeline id creates a new consumer group automatically? What is a consumer group and why do we need to make new ones?

Reply:
A consumer group (in our simple case with one consumer in the group) is a set of references to topic offsets which the group has read. One can manually change the current offset of a consumer group to an arbitrary offset. A newly created consumer group will start at an offset which is determined by the consumer's policy, basically either starting from the beginning, or the next message to be posted to the topic and then continue sequentially from there as more data becomes available.

Reply:
I like to version pipelines just so I know what's been updated, but you're right that according to this:
https://github.com/eHealthAfrica/aether-stream-consumer/blob/f45df359f79a3bbb4ed20c30f0e39df18b35d149/stream-consumer/app/artifacts.py#L188
It should create a new consumer group, which according to the current settings:
https://github.com/eHealthAfrica/concourse-pipelines/blob/5306212233ae34e5c0c21eef4a38b8f0fb184ea0/aether/prod/overrides/common-consumer.yaml#L56
Should actually start from the first offset. I think the reason we haven't had a bunch of replayed messages is that topic name (and topic pattern) has changed recently.